kore/src/Kore/JsonRpc.hs

```diff
@@ -757,10 +755,7 @@ runServer port serverState mainModule runSMT Log.LoggerEnv{logAction} = do
     flip runLoggingT logFun $
         jsonRpcServer
             srvSettings
-            ( \req parsed ->
-                log (InfoJsonRpcProcessRequest (getReqId req) parsed)
```

Comment:
we already emit a similar log message on receiving requests in `Kore.JsonRpc.Server.srv`